mldsa/fips202/fips202.c

```diff
@@ -562,6 +562,45 @@ __contract__(
   s[(r - 1) / 8] ^= 1ULL << 63;
 }
 
+/*************************************************
+ * Name:        keccak_squeezeblocks
+ *
+ * Description:
+ *
+ * Arguments:   - uint8_t *out: pointer to output blocks
+ *              - size_t nblocks: number of blocks to be squeezed (written to
+ *out)
+ *              - uint64_t *s: pointer to input/output Keccak state
+ *              - unsigned int r: rate in bytes (e.g., 168 for SHAKE128)
+ **************************************************/
```

Comment:
This documentation needs updating

Reply:
Sorry, I got lost between multiple github branches! I just updated it.